steps/infisical-secrets/0.0.2/step.yml

```diff
@@ -0,0 +1,65 @@
+title: Infisical Secrets
+summary: |
+  Pull secrets from an Infisical secrets repository
+description: |
+  Use this step to access your cloud or self hosted Infisical secrets and inject them into your build pipeline.
```

Comment:
NOTE: It can be useful to link to the integrated services home page in the Steps description.